ezbake/config/logback.xml

```diff
@@ -18,6 +18,7 @@
 
     <root level="info">
         <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="${logappender:-DUMMY}"/>
```

Comment:
Wow.  Is that BASH style syntax for logappender taking on the value DUMMY if it's not already set?